lib/served/json_api_error/error.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'active_support/core_ext/hash/indifferent_access'
+module Served
+  module JsonApiError
```

Comment:
lets namespace this under `Serializers::JsonApi::Error` as it relates specifically to the serializer